internal/services/apimanagement/api_management_logger_resource.go

```diff
@@ -70,10 +70,20 @@ func resourceApiManagementLogger() *pluginsdk.Resource {
 
 						"connection_string": {
 							Type:         pluginsdk.TypeString,
-							Required:     true,
+							Optional:     true,
 							Sensitive:    true,
 							ValidateFunc: validation.StringIsNotEmpty,
 						},
+						"endpoint_address": {
+							Type:         pluginsdk.TypeString,
+							Optional:     true,
+							ValidateFunc: validation.StringIsNotEmpty,
```

Comment:
The the `AtLeastOneOf` attribute needs to be added to `connection_string` as well
```suggestion
							ValidateFunc: validation.StringIsNotEmpty,
							AtLeastOneOf: []string{
								"eventhub.0.connection_string",
								"eventhub.0.endpoint_uri",
							},
```